examples/exchanger.py

```diff
@@ -919,6 +922,7 @@ def localized_arg(bytestring):
 
     list_tables = nspi_attacks.add_parser('list-tables', help='List Address Books')
     list_tables.add_argument('-count', action='store_true', help='Request total number of records in each table')
+    list_tables.add_argument('-basic', action='store_true', help='Authenticate with Basic Auth instead of NTLM')
```

Comment:
The argument is only available for nspi list-tables, shouldn't it be moved out of it and into the general flags for this module? (could move it below '-rpc-hostname' argument)
```suggestion
    parser.add_argument('-basic', action='store_true', help='Authenticate with Basic Auth instead of NTLM')
```